packages/back-end/src/integrations/Snowflake.ts

```diff
@@ -43,60 +34,14 @@ export default class Snowflake extends SqlIntegration {
   ensureFloat(col: string): string {
     return `CAST(${col} AS DOUBLE)`;
   }
-  async getInformationSchema(): Promise<InformationSchema[]> {
-    const database = this.params.database;
-
-    if (!database) {
+  getInformationSchemaFromClause(): string {
+    if (!this.params.database)
       throw new Error(
         "No database provided. In order to get the information schema, you must provide a database."
       );
-    }
-
-    const queryString = `SELECT
-        table_name,
-        table_catalog,
-        table_schema,
-        count(column_name) as column_count
-    FROM
-        ${database}.INFORMATION_SCHEMA.COLUMNS
-    WHERE
-        table_schema
-      NOT IN ('INFORMATION_SCHEMA')
-    GROUP BY (table_name, table_schema, table_catalog)`;
-
-    const results = await this.runQuery(queryString);
-
-    if (!results.length) {
-      throw new Error(`No tables found for database "${database}".`);
-    }
-
-    return formatInformationSchema(
-      results as RawInformationSchema[],
-      "snowflake"
-    );
+    return `${this.params.database}.information_schema.columns`;
   }
-
-  async getTableData(
-    databaseName: string,
-    tableSchema: string,
-    tableName: string
-  ): Promise<{ tableData: null | unknown[]; refreshMS: number }> {
-    const sql = `SELECT
-          data_type,
-          column_name
-        FROM
-          ${databaseName}.INFORMATION_SCHEMA.COLUMNS
-        WHERE
-          table_name
-        IN ('${tableName}')
-        AND
-          table_schema
-        IN ('${tableSchema}')`;
-
-    const queryStartTime = Date.now();
-    const tableData = await this.runQuery(sql);
-    const queryEndTime = Date.now();
-
-    return { tableData, refreshMS: queryEndTime - queryStartTime };
+  getInformationSchemaTableFromClause(databaseName: string): string {
+    return `${databaseName}.INFORMATION_SCHEMA.COLUMNS`;
```

Comment:
In general, I've noticed in all of these, we're doing lowercase for the top-level query, then uppercase for the table query.  We should make them consistent.

Reply:
I've made the default lowercase, and only added uppercasing when needed (for BigQuery and Snowflake). I made a comment in-line with an explanation.